src/components/List.js

```diff
@@ -5,14 +5,15 @@ import Contact from './Contact';
 class List extends React.Component {
 
   get items() {
-    const { array, text } = this.props;
+    const { array, methods, text } = this.props;
```

Comment:
Use more specific naming (contacts or contactsList instead of array) (contectHandlers instead of methods). And not sure what 'text' reffers to. Is it just the type of the contact?